lib/email/store.rb

```diff
@@ -0,0 +1,38 @@
+# move this require in gemspec
+require 'dalli'
+
+class Email::Store
+  MEMCACHED_NAMESPACE = 'email_store'
```

Comment:
Freeze mutable objects assigned to constants.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.